aruba.gemspec

```diff
@@ -21,9 +19,21 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'rspec-expectations', '~> 3.4'
   spec.add_runtime_dependency 'thor', '~> 0.19'
 
-  spec.add_development_dependency 'bundler', ['>= 1.7.0', '< 3.0']
+  spec.add_development_dependency 'fuubar', '~> 2.3'
+  spec.add_development_dependency 'json', '~> 2.1'
+  spec.add_development_dependency 'license_finder', '~> 5.3'
+  spec.add_development_dependency 'minitest', '~> 5.10'
+  spec.add_development_dependency 'pry-doc', '~> 1.0.0'
+  spec.add_development_dependency 'rake', '~> 12.3'
+  spec.add_development_dependency 'rspec', '~> 3.6'
+  spec.add_development_dependency 'rubocop', '~> 0.69'
```

Comment:
This should be `~> 0.69.0` because the set of cops can change with each release.

Reply:
I agree with this, but I just copied the existing config, all this PR is aimed to be is a tidy up.